dataprocessors/json/observation_point.go

```diff
@@ -0,0 +1,77 @@
+package json
+
+import (
```

Comment:
How much of this file can go into the main project? The marshal/unmarshal time seems like it can be re-used from utils and instead of re-defining Observation here, could we use a modified `observations.Observation`?